### DISCUSSION TITLE
The same plot gives different results

### DISCUSSION THREAD

**Discussion by KmBooy** [Chia-Network/chia-blockchain#6193] [Support]
![image](https://user-images.githubusercontent.com/35790216/120052430-e31dc980-c057-11eb-9d53-48a066de5dd9.png)
Run “chia plots check” get the results .
The same plot have different results

**Comment by tiny0little** [comment]
could you post output of these two commands
`chia keys show | egrep 'Pool|Farm'`
`chia plots check -n 30 -g 2021-05-28-02-43`
remove `Local sk: <PrivateKey` line from last output

**Comment by BrandtH22** [comment]
@KmBooy , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it is expected that running a plot check multiple times will yield different results, this is due to the probabilistic nature of finding proofs. The greater concern from the image is the plots that do not have an associated farmer key. Unless this is a harvester that is connected to a farmer that does have those keys you may need to replot those plots.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
